lib/ex_prosemirror/block/hard_break.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ExProsemirror.Block.HardBreak do
+  @moduledoc ~S"""
+  Hard Break tag
+  """
+
+  use ExProsemirror.Schema
+  use ExProsemirror.Encoder.Json, type: :hard_break
+
+  import Ecto.Changeset
+
+  @doc false
+  embedded_schema do
+  end
+
+  @doc false
+  def changeset(struct_or_changeset, attrs \\ %{}) do
+    struct_or_changeset
+    |> cast(attrs, [])
```

Comment:
I don't think you need to cast as there is nothing in the schema.

```suggestion
```

Reply:
this cast is needed to generate a changeset